packages/make-styles/src/makeStyles.ts

```diff
@@ -0,0 +1,85 @@
+import { CAN_USE_CSS_VARIABLES, DEFINITION_LOOKUP_TABLE } from './constants';
+import { createCSSVariablesProxy, resolveDefinitions } from './runtime/index';
+import {
+  MakeStylesDefinition,
+  MakeStylesMatchedDefinitions,
+  MakeStylesOptions,
+  MakeStylesResolvedDefinition,
+} from './types';
+
+export function makeStyles<Selectors, Tokens>(definitions: MakeStylesDefinition<Selectors, Tokens>[]) {
+  const cxCache: Record<string, string> = {};
+
+  return function computeClasses(
+    selectors: Selectors,
+    options: MakeStylesOptions<Tokens>,
```

Comment:
Where am I supposed to take `options`? Is the plan to create a wrapper which will take the renderer and other options from context?

Reply:
This package represents low level API, for exported API I propose to have bindings, for example:

```ts
export function makeStyles(definitions) {
  const result = vanillaMakeStyles(definitions);

  return function ___(selectors, ...classNames): string {
    const { rtl, theme, makeStylesRenderer } = React.useContext(FluentContext);
    const options = {
      rtl,
      tokens: theme.siteVariables,
      renderer: makeStylesRenderer
    };

    return result(selectors, options, ...classNames);
  };
}
```